bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/contract/form/helper/FormType.java

```diff
@@ -0,0 +1,16 @@
+package org.bahmni.module.bahmnicore.contract.form.helper;
```

Comment:
Why create additional package like helper within the contract package?  FormType is an enum, and just putting under the "org.bahmni.module.bahmnicore.contract.form" package should suffice, wouldn't it?